keps/1136-provisioning-request-support/README.md

```diff
@@ -95,11 +95,13 @@ The `ProvisioningRequest` should have the owner reference set to the workload.
 To understand what details should it put into `ProvisioningRequest` the controller
 will also need to watch `ProvisioningRequestConfigs`.
 
-* Watch all changes CA makes to `ProvisioningRequests`. If the `Provisioned`
-or `CapacityAvailable` condition is set to `True` then finish the `AdmissionCheck`
-with success (and propagate the information about `ProvisioningRequest` name to
+* Watch all changes CA makes to `ProvisioningRequests`. If the `ProvisioningRequest's` conditions are set to:
+  - `Provisioned=false` controller should surface information about ProvisioningRequest's ETA. It should emit an event regarding that and for every ETA change.
+  - `Provisioned=true` controller should mark the AdmissionCheck as `Ready` and propagate the information about `ProvisioningRequest` name to
 workload pods - [KEP #1145](https://github.com/kubernetes-sigs/kueue/blob/main/keps/1145-additional-labels/kep.yaml) under `"cluster-autoscaler.kubernetes.io/consume-provisioning-request"`.
-If the `ProvisioningRequest` fails, fail the `AdmissionCheck`.
+  - `Failed=true` controller should retry AdmissionCheck with respect to the `RetryConfig` configuration, or mark the AdmissionCheck as `Rejected`
+  - `BookingExpired=true` controller should check if `ProvisioningRequest` needs recreating - when the provisioned nodes have no pods scheduled, and retry if needed with respect to the `RetryConfig` configuration.
```

Comment:
I don't think we need to check provisioned nodes. For example for Check-capacity ProvisioningClass there are no provisioned nodes. 
I think ProvisioningRequest should be recreated if the pods weren't created yet or if the pods were created but not all of them were scheduled during some time.

Reply:
+1 on an even simpler approach of checking if the job is unsuspended.

Maybe start a section in Risks and Mitigations to describe when the race condition that could happen. IIRC, it could only happen if the user has more than one admission check or when creating a massive job, where the job controller is not able to create pods fast enough
For the first case, we have WaitForPodsReady as a possible mitigation (after the timeout, the job is evicted and retried).
For the second case, the cluster administrator should probably size their control plane appropriately (bigger VMs and/or bigger qps for the controller creating pods).